Testing/functional/tests/cypress/integration/e2e/vaccineCard/publicSave.js

```diff
@@ -36,6 +36,61 @@ function select(selector, value) {
     cy.get(selector).should("be.visible", "be.enabled").select(value);
 }
 
+describe("Public User - Vaccine Card Page - Save with Retry", () => {
+    beforeEach(() => {
+        deleteDownloadsFolder();
+        let isLoading = false;
+        cy.intercept("GET", "**/v1/api/PublicVaccineStatus/pdf", (req) => {
+            if (!isLoading) {
+                req.reply({
+                    fixture: "ImmunizationService/vaccineProofNotLoaded.json",
+                });
+            } else {
+                req.reply({
+                    fixture: "ImmunizationService/vaccineProofLoaded.json",
+                });
+            }
+            isLoading = !isLoading;
+        });
+    });
+    it("Save Dropdown List - Save PDF - Download confirmed With Retry", () => {
+        const phn = "9735361219 ";
+        const dobYear = "1994";
+        const dobMonth = "June";
+        const dobDay = "9";
+        const dovYear = "2021";
+        const dovMonth = "January";
+        const dovDay = "20";
+        cy.enableModules([
+            "Immunization",
+            vaccinationStatusModule,
+            "VaccinationStatusPdf",
+            "PublicVaccineDownloadPdf",
+        ]);
+        cy.visit(vaccineCardUrl);
+        enterVaccineCardPHN(phn);
+        select(dobYearSelector, dobYear);
+        select(dobMonthSelector, dobMonth);
+        select(dobDaySelector, dobDay);
+        select(dovYearSelector, dovYear);
+        select(dovMonthSelector, dovMonth);
+        select(dovDaySelector, dovDay);
+        clickVaccineCardEnterButton();
+        cy.get("[data-testid=save-dropdown-btn]")
+            .should("be.visible", "be.enabled")
+            .click();
+        cy.get("[data-testid=save-as-pdf-dropdown-item]")
+            .should("be.visible")
+            .click();
+        cy.get("[data-testid=genericMessageModal]").should("be.visible");
+        cy.get("[data-testid=genericMessageSubmitBtn]").click();
+        cy.get("[data-testid=loadingSpinner]").should("be.visible");
+        cy.wait(1000);
```

Comment:
The fixture has a "retryin" value of 10000, so we should wait the same amount of time in the test before checking for changes.